loch/lxLRUD.h

```diff
@@ -21,7 +21,6 @@ struct lxLRUDShot {
   long xfrom = 0, xto = 0;
   int profile = LXLRUD_OCTAGON;
   double fl = 0.0, fr = 0.0, fu = 0.0, fd = 0.0, tl = 0.0, tr = 0.0, tu = 0.0, td = 0.0, vthresh = 60.0;
-  void * usr_data = nullptr;
```

Comment:
Aj toto sa moze zist. lxLRUD bolo navrhnute pomerne univerzalne, naozaj to tam treba nechat.